field_utils/tests/Test_FieldCondensedArray_private.pf

```diff
@@ -0,0 +1,147 @@
+#include "MAPL_TestErr.h"
+module Test_FieldCondensedArray_private
+
+    use MAPL_ExceptionHandling
+    use pfunit
+    use mapl3g_FieldCondensedArray_private
+    implicit none
+
+    character, parameter :: GENERIC_MESSAGE = 'actual does not match expected.'
+    
+contains
+
+    @Test
+    subroutine test_get_array_shape_3D()
+       integer :: expected(3), actual(3)
+       integer, allocatable :: gridToFieldMap(:)
+       integer, allocatable :: localElementCount(:)
+       integer, allocatable :: vertical_dims(:)
+
+       gridToFieldMap = [1, 2]
+       localElementCount = [3, 5, 7]
```

Comment:
This localElementCount is inconsistent with the specification on the line below it.   Makes the test suspect.